filters/imagefilter.go

```diff
@@ -101,3 +103,10 @@ func transformImage(out *io.PipeWriter, image *bimg.Image, opts *bimg.Options) e
 
 	return err
 }
+
+func applyDefaults(o *bimg.Options) *bimg.Options {
+	if o.Quality == 0 {
```

Comment:
Right, but I think that every library has a default quality. I think they only expect to perform all the operation on the image in 1 call.